src/cli/build.rs

```diff
@@ -35,6 +37,10 @@ pub struct Args {
     /// The output directory to place the build artifacts
     #[clap(long, short, default_value = ".")]
     pub output_dir: PathBuf,
+
+    /// Whether to build incrementally if possible
+    #[clap(long, short, default_value_t = true)]
+    pub incremental: bool,
```

Comment:
```suggestion
    #[clap(long, short, default_value_t = true)]
    pub incremental: bool,
```

Default value true doesn't make a lot of sense according to my understanding for flags. There's no way to disable them.

Common practice is `--no-incremental`

Reply:
Makes sense, changed it